restli-server/src/main/java/com/linkedin/restli/internal/server/response/ResponseUtils.java

```diff
@@ -97,7 +117,19 @@ public static RestResponse buildResponse(RoutingResult routingResult, RestLiResp
                                                          .getResourceEntityType();
     if (restLiResponse.hasData() && ResourceEntityType.STRUCTURED_DATA == resourceEntityType)
     {
+      DataSchema dataSchema = null;
+      Class<? extends RecordTemplate> valueClass = routingResult.getResourceMethod().getResourceModel().getValueClass();
+      if (valueClass != null)
+      {
+        dataSchema = DataTemplateUtil.getSchema(valueClass);
+      }
+
       DataMap dataMap = restLiResponse.getDataMap();
+      if (context.getParameters().containsKey("sendDefaults") &&
```

Comment:
$sendDefaults, doc was updated to add $ as prefix to not conflict with other fields.
Also, can you move it to as constant instead of hard coding here

Reply:
this is updated